src/main/java/by/onliner/page/pages/ShoppingCartPage.java

```diff
@@ -37,13 +37,15 @@ public void waitForPageOpened() {
     public void removeItemFromCart() {
         Waiter.waitForVisibility(removeFromCartButton);
         ActionsHelper.moveToElementAndClick(removeFromCartButton);
+        logger.info("Click 'remove from cart' button");
     }
 
     /***
      * Get a message from removed item.
      */
     public String getRemovedItemInformation() {
         Waiter.elementToBeClickable(removedItemInformation);
+        logger.info(String.format("Displayed ...%s... message", removedItemInformation.getText()));
```

Comment:
Why do you need ... in the message? you can just use brackets([ ]) or smth like that, it will look leaner

Reply:
Fixed